nemo/collections/asr/parts/jasper.py

```diff
@@ -21,6 +21,18 @@
 
 from nemo.collections.asr.parts.activations import Swish
 
+try:
+    from pytorch_quantization import calib
+    from pytorch_quantization import nn as quant_nn
+    from pytorch_quantization import quant_modules
+    from pytorch_quantization.tensor_quant import QuantDescriptor
+except ImportError:
```

Comment:
Needs to be an optional import under a try catch guard
```python
try:
    from pytorch_quantization import calib
    from pytorch_quantization import nn as quant_nn
    from pytorch_quantization import quant_modules
    from pytorch_quantization.tensor_quant import QuantDescriptor
    PYTORCH_QUANTIZATION_AVAILABLE = True
except (ImportError, ModuleNotFoundError):
    PYTORCH_QUANTIZATION_AVAILABLE  = False
```

After this, every `if quantize` flag must be replaced with `if PYTORCH_QUANTIZATION_AVAILABLE  and quantize:`